plugins/shell-ctx.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: shell-ctx
+spec:
+  version: "v1.0.2"
+  homepage: https://github.com/glemsom/shell-ctx
+  shortDescription: "Kubernetes shell isolated context switcher"
```

Comment:
I think this would be a higher signal:

> Launch isolated shell for context

what do you think?

Reply:
I see the point of the current shortDescription being slightly misleading.
But `shell-ctx` does actually not spawn a new shell.

What about:
> Shell independent context switching

Will that be better ?

Clarification;
The basic idea behind `shell-ctx` is to hook into the users current shell-instance - allowing the user to keep the same current shell-history,  and any other shell-specific settings the user might have.
This will enable Kubernetes administrators to have multible shell windows open, and switch Kubernetes context per shell instance - without risking overwriting currentContext of any other shell instances.

Reply:
Ah yeah makes sense.